src/main/app/cypress/integration/komoto/osaamisala.spec.js

```diff
@@ -0,0 +1,22 @@
+const autoRecord = require('cypress-autorecord');
+
+describe('Osaamisala KOMOTO', function () {
+  autoRecord();
+
+  it('Osaamisala KOMOTO renders properly', function () {
+    cy.visit('/konfo/fi/toteutus/1.2.246.562.17.00000000000000000471');
+
+    // Wait for everything to load
+    cy.findByRole('progressbar').should('not.exist');
+    cy.get('h1').contains('Hevosten kengittämisen osaamisala');
+    cy.contains('100 osaamispistettä');
+    cy.get('h2').contains('Ilmoittaudu koulutukseen');
+    cy.get('a[href="http://www.google.fi"]').contains('Ilmoittaudu koulutukseen');
```

Comment:
Mä tekisin näin, koska tällöin query muistuttaa näin enemmän tapaa, jolla käyttäjä löytää kyseisen elementin.

```suggestion
    cy.findByRole('link', {name: /ilmoittaudu koulutukseen/i}).should('have.attr', 'href', 'http://www.google.fi');
```